changelog/1.17.2.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "1.17.2"
+description: "Released on 03/30/21"
+---
+
+### Breaking Changes ❗
+
+There are no breaking changes in 1.17.2.
+
+### Features ✨
+
+- Workspaces As Code (**WAC**): Workspaces as code (WAC) brings the
+  _Infrastructure As Code_ paradigm to Coder environments. WAC allows you to
+  define and create new environments using **workspace templates**.
+  - web: New page available via **Admin** > **Templates** for creating an
+    embeddable quickstart button.
+  - web: New options available when clicking **New Environment** from the
+    **Environments** page
+- web: Added _Cordon_ and _Uncordon_ actions to _Provider_ audit logs
+
+### Bug Fixes 🐛
+
+- infra: Fixes an issue whereby using `coder/configure` to create Dev URLs would
+  fail
+- infra: Fixes an issue whereby authenticating with GitLab OIDC may fail if name
```

Comment:
The problem is that GitLab's OIDC does not return a name at all, so I would suggest phrasing as

```suggestion
- infra: Fixes an issue authenticating using OpenID Connect Identity Providers (IdPs) that omit name information, such as GitLab. In this case, Coder will use the email address as the user's name.
```